src/pages/givingform.tsx

```diff
@@ -0,0 +1,834 @@
+                                                <option value="other">
+                                                    Other
+                                                </option>
+                                            </select>
+                                        </div>
+                                        {showOtherFrequencyInput ? (
+                                            <div
+                                                className={
+                                                    formStyles.formItemSingle
+                                                }
+                                            >
+                                                <label
+                                                    className={
+                                                        formStyles.formItemLabel
+                                                    }
+                                                >
+                                                    Commencing from
+                                                </label>
+                                                <input
+                                                    type="text"
+                                                    className={
+                                                        formStyles.formItemInput
+                                                    }
+                                                    placeholder={
+                                                        "Other Frequency"
+                                                    }
+                                                    name="otherRegularGiftRequency"
+                                                    ref={register}
+                                                />
+                                            </div>
+                                        ) : (
+                                            <></>
+                                        )}
+                                        <div
+                                            className={
+                                                formStyles.formItemSingle
+                                            }
+                                        >
+                                            <label
+                                                className={
+                                                    formStyles.formItemLabel
+                                                }
+                                            >
+                                                Commencing from
+                                            </label>
+                                            <input
+                                                className={
+                                                    formStyles.formItemInput
+                                                }
+                                                type="date"
```

Comment:
I've recall hearing cross-browser support for `<input type="date" />` is a bit patchy so let's test this on a few devices before rolling out, otherwise we might have to grab a date picker off npm.